packages/dd-trace/src/appsec/iast/taint-tracking/operations-taint-object.js

```diff
@@ -4,6 +4,7 @@ const TaintedUtils = require('@datadog/native-iast-taint-tracking')
 const { IAST_TRANSACTION_ID } = require('../iast-context')
 const iastLog = require('../iast-log')
 
+// @see packages/dd-trace/test/appsec/iast/taint-tracking/taint-tracking-operations.spec.js
 function taintObject (iastContext, object, type, keyTainting, keyType) {
```

Comment:
why aren't we removing `keyTainting, keyType` from the method? If it is not working well, why maintain unused code if we can clean it up?